src/interface/core/lang_config.ts

```diff
@@ -31,53 +31,19 @@ function setFallback(notFoundLang: string): void {
 StaticConfig.keyChanged('appLanguage', (newLanguage: string) => {
 	if (Languages[newLanguage].name === 'Default') {
 		// Get the language part from the locale (e.g. en-GB -> en)
-		const locale = globalAny.navigator.language.substr(0, globalAny.navigator.language.indexOf('-'))
+		const newLocale = globalAny.navigator.language.substr(0, globalAny.navigator.language.indexOf('-'))
```

Comment:
I think it would be better to use this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale

Example: 
```ts
const { language } = new Intl.Locale(globalThis.navigator.language)
```

Reply:
Using `Intl.Locale` returns `Property 'Locale' does not exist on type 'typeof Intl'.` Do you have any solutions for this in mind?

Reply:
> Using `Intl.Locale` returns `Property 'Locale' does not exist on type 'typeof Intl'.` Do you have any solutions for this in mind?

Where did you test it? It worked fine for me.

Reply:
> Where did you test it? It worked fine for me.
I am currently testing on Ubuntu 20.10.

Reply:
> 
> 
> > Where did you test it? It worked fine for me.
> 
> I am currently testing on Ubuntu 20.10.

I mean, where are you executing that code? Node? Electron? Firefox?

Reply:
> I mean, where are you executing that code? Node? Electron? Firefox?

Electron.

Reply:
> 
> 
> > I mean, where are you executing that code? Node? Electron? Firefox?
> 
> Electron.

Show me the error, please.

Reply:
Here is the console output:
```bash
↳✨ Watching in WatchWebpackRenderer
   ↳❌ [webpack] Error in: ./src/interface/core/lang_config.ts

 [tsl] ERROR in /home/user/Coding/Graviton-App/src/interface/core/lang_config.ts(33,33)
      TS2339: Property 'Locale' does not exist on type 'typeof Intl'.

```
And line 33:
```Typescript
const { language } = new Intl.Locale(globalThis.navigator.language)
```

Also, using `(Intl as any).Locale(...)` works normally.

Reply:
> 
> 
> Here is the console output:
> 
> ```shell
> ↳✨ Watching in WatchWebpackRenderer
>    ↳❌ [webpack] Error in: ./src/interface/core/lang_config.ts
> 
>  [tsl] ERROR in /home/user/Coding/Graviton-App/src/interface/core/lang_config.ts(33,33)
>       TS2339: Property 'Locale' does not exist on type 'typeof Intl'.
> ```
> 
> And line 33:
> 
> ```ts
> const { language } = new Intl.Locale(globalThis.navigator.language)
> ```
> 
> Also, using `(Intl as any).Locale(...)` works normally.

Related to https://github.com/microsoft/TypeScript/issues/37326 .

I think it would be better to just leave it as it is for now